common/components/notices/BusDataNotice.tsx

```diff
@@ -1,17 +1,22 @@
 import React from 'react';
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
+import { faMagnifyingGlassChart } from '@fortawesome/free-solid-svg-icons';
 import { useDelimitatedRoute } from '../../utils/router';
 
 export const BusDataNotice: React.FC = () => {
   const { line, linePath } = useDelimitatedRoute();
 
   if (line === 'BUS' || linePath === 'bus') {
     return (
-      <div className={'m-3 text-sm italic'}>
-        <p>
-          Due to data collection issues, bus data is not guaranteed to be complete for any stop or
-          date.
-        </p>
-        <p>This may lead to inaccuracies, particularly in headway calculations.</p>
+      <div className={'flex items-center'}>
+        <FontAwesomeIcon icon={faMagnifyingGlassChart} size={'2x'} />
```

Comment:
Maybe a little smaller?
![Screen Shot 2023-04-07 at 2 17 36 PM](https://user-images.githubusercontent.com/46229349/230657946-870fa41a-7ea8-49b4-ab26-9b4e316a7c44.png)

```suggestion
        <FontAwesomeIcon icon={faMagnifyingGlassChart} className="h-6 w-6" />
```